app/assets/javascripts/admin/addon/components/admin-config-area-cards/about/contact-information.gjs

```diff
@@ -72,12 +150,22 @@ export default class AdminConfigAreasAboutContactInformation extends Component {
       <p class="admin-config-area-card__additional-help">
         {{i18n "admin.config_areas.about.site_contact_group_help"}}
       </p>
-      <GroupChooser />
+      <GroupChooser
+        @content={{this.site.groups}}
+        @value={{this.contactGroupId}}
+        @onChange={{this.onContactGroupIdChange}}
+        @options={{hash maximum=1}}
+      />
     </div>
     <DButton
       @label="admin.config_areas.about.update"
       @action={{this.save}}
-      class="btn-primary"
+      class="btn-primary save-card"
```

Comment:
Let's make this more specific for config areas, should be `admin-config-area-card__btn-save`